CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Decidim::Blogs::Post.find_each(&:add_to_index_as_search_resource)
 
 **Changed**:
 
+- **many modules**: Improved visiblity of buttons: new proposal, debate and initiative. [\#5535](https://github.com/decidim/decidim/pull/5535)
```

Comment:
I'd better name the modules affected, such as:

-  **decidim-proposals**, **decidim-debates** and **decidim-initiatives**

Reply:
Like this?
https://github.com/decidim/decidim/pull/5535/commits/4b30c9c9c85f088e662ab100bd93a10ded68302e#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR24

Reply:
yes